package.json

```diff
@@ -1,13 +1,15 @@
 {
-  "name": "d2l-typography",
   "description": "Polymer and Sass custom styles/mixins for applying D2L typography",
-  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/typography.git"
+  },
+  "name": "d2l-typography",
   "scripts": {
     "build:sass": "node-sass --output-style expanded ./test/acceptance/typography.css.scss ./test/acceptance/typography.css",
-    "postinstall": "polymer install --variants",
-    "test:lint": "npm run test:lint:wc && npm run test:lint:html",
-    "test:lint:html": "eslint *.html",
-    "test:lint:wc": "polymer lint -i *.html",
+    "test:lint": "npm run test:lint:wc && npm run test:lint:js",
+    "test:lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
+    "test:lint:wc": "polymer lint",
```

Comment:
Will need to add `"pretest:lint": "npm run build:sass"` in order to successfully use `polymer lint` on the test folder.